Hansard.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
+
+# download the data from TheyWorkForYou by using:
+# rsync -az --exclude='*s19*'  --progress --exclude '.svn' --exclude 'tmp/'
+# --relative  data.theyworkforyou.com::parldata/scrapedxml/debates/  .
+
+# load libraries
+import os
+import re
+
+import pandas as pd
+
+from lxml import etree  # nosec B3410
+
+
+# --------------------------------------------------------------------------------------------------------------------
+# Define functions:
+def FilePaths(directory: str) -> list:
```

Comment:
It's great that you're using type hints!

As a next step, you can use the `typing` library
```python
from typing import List
```

and then define more precisely what kind of elements this list will contain, eg:

```suggestion
def FilePaths(directory: str) -> List[str]:
```

Find more information here: https://docs.python.org/3/library/typing.html